src/rascal/representations/calculator_spherical_invariants.hh

```diff
@@ -216,9 +265,168 @@ namespace rascal {
         this->compute_gradients = false;
       }
 
+      this->max_radial = hypers.at("max_radial").get<size_t>();
+      this->max_angular = hypers.at("max_angular").get<size_t>();
+
+      if (hypers.find("coefficient_subselection") != hypers.end() and
```

Comment:
Is the format of `coefficient_subselection` documented somewhere? In fact, this whole function needs documentation (docstring, not more inline comments). Since this PR adds only the coefficient subselection to this function, I would accept just the documentation of that parameter. It is not easy to figure out from the code -- or the inline comments -- how this is supposed to be formatted or what utilities there are to generate such a selection.

Reply:
Ok, missed the documentation that was added to the Python bindings. It's still a good idea to have some documentation here, especially about the constraints on sorting and not exceeding `n_max` and `l_max` (_not_ in inline comments, those aren't visible in Doxygen).

Reply:
added comment